backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/computing/purchasecontract/subtractedvalue/interim/InterimInvoiceLineLog.java

```diff
@@ -64,4 +64,10 @@ protected int getMultiplier(final @NonNull CreateLogRequest createLogRequest)
 		Check.assumeNotNull(priceActual, "PriceActual shouldn't be null");
 		return priceActual.negate();
 	}
+
+	@Override
+	protected @NonNull ProductPrice getPriceActual(final @NonNull CreateLogRequest request)
+	{
+		return super.getPriceActual(request).negate();
```

Comment:
why do we have to negate the price?
why did the price come negative?

IMHO if we fix it in this way, I will it is very fragile and will break any time soon.

cc @metas-rc

Reply:
The initial price is positive (from contract specific prices), we negate if costType and we must additionally negate because of subtracted value.
Also this need to be in sync with function above `de.metas.contracts.modular.computing.purchasecontract.subtractedvalue.interim.InterimInvoiceLineLog#getPriceActual(ModularContractLogEntry logEntry)`

Maybe we could consider to initiate the contract specific prices for subtracted values with negative prices, but I think this would lead to confusion now.

For InterimInvoiceLine this isn't enough, because getPriceActual isn't used on createLog. See getMultiplier on abstract interim

Reply:
pls also create an dev test for this, feels like this isn't correct after